presto-druid/src/main/java/com/facebook/presto/druid/DruidSplitManager.java

```diff
@@ -21,21 +28,31 @@
 import com.facebook.presto.spi.connector.ConnectorSplitManager;
 import com.facebook.presto.spi.connector.ConnectorTransactionHandle;
 import com.google.common.collect.ImmutableList;
+import io.airlift.slice.Slice;
 
 import javax.inject.Inject;
 
+import java.time.Instant;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeFormatterBuilder;
+import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 
 import static com.facebook.presto.druid.DruidSessionProperties.isComputePushdownEnabled;
 import static com.facebook.presto.druid.DruidSplit.createBrokerSplit;
 import static com.facebook.presto.druid.DruidSplit.createSegmentSplit;
 import static com.google.common.collect.ImmutableList.toImmutableList;
+import static java.lang.String.format;
 import static java.util.Objects.requireNonNull;
 
 public class DruidSplitManager
         implements ConnectorSplitManager
 {
     private final DruidClient druidClient;
+    // Druid stores segments times as strings in ISO-8601 format in UTC down to fractional seconds.
```

Comment:
do we need to consider the timezone in presto when filtering on __time?

Reply:
Its always UTC, so I think I need to put that in the comments and variable names to make it clear.